tests/npltests/npl_pod_test.go

```diff
@@ -51,7 +51,7 @@ const (
 	defaultHostIP   = "10.10.10.10"
 	defaultPodIP    = "192.168.32.10"
 	defaultPodPort  = 80
-	defaultNodePort = 40001
+	defaultNodePort = 61000
```

Comment:
Should this port be changed back to 4x/5x as 1.10.5 might not have support for ports in 6x range?
cc @akshayhavile

Reply:
@pkoshtavmware the npl port range should not be a problem in UT. Also, 61000-62000 and previously 4x/5x  is only the default range and the user can still specify a different port range so port value may not necessarily be in that range.